pkg/cmd/kafka/create/create.go

```diff
@@ -206,6 +206,8 @@ func runCreate(opts *options) error {
 	a = a.KafkaRequestPayload(*payload)
 	a = a.Async(true)
 
+	return nil
```

Comment:
Accidental?

Reply:
This is for testing without creating +5 kafkas. I will remove it later. But you can replicate that yourself locally and removing it now

Reply:
Removed